mlflow/tracking/fluent.py

```diff
@@ -1632,7 +1660,11 @@ def setup_autologging(module):
             if prev_config and not prev_config.get(CONF_KEY_IS_GLOBALLY_CONFIGURED, False):
                 return
 
-            autologging_params = get_autologging_params(autolog_fn)
+            autologging_params = (
+                _get_tensorflow_autologging_params()
+                if module_name == "tensorflow"
+                else get_autologging_params(autolog_fn)
+            )
```

Comment:
For maintainability, it would be nice if we could scope this special-case logic inside the `mlflow.tensorflow.autolog()` module. One way to do this would be to leverage the value of `AUTOLOGGING_INTEGRATIONS["tensorflow"].get(CONF_KEY_IS_GLOBALLY_CONFIGURED)`, similar to https://github.com/mlflow/mlflow/blob/855881f93703b15ffe643003fb4d7c84f0ec2502/mlflow/tracking/fluent.py#L1631-L1632.

During TF autologging, we can check to see if autologging for TensorFlow is globally configured and, if so, default `log_model_signatures` to `False`.

Reply:
Implemented this here: https://github.com/mlflow/mlflow/pull/5652. Feel free to rebase / merge master once #5652 is merged.